CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [2.3.6] - 2019-01-24
```

Comment:
We don't fill in the number or date until we know for sure what these are going to be, so you can set these both to "TBD" for now.

Reply:
Done, thanks